src/ClearingHouse.sol

```diff
@@ -249,15 +270,7 @@ contract ClearingHouse is Policy, RolesConsumer, CoolerCallback {
     function defund(ERC20 token_, uint256 amount_) external onlyRole("cooler_overseer") {
         if (token_ == gOHM) revert OnlyBurnable();
 
-        // If there is an outstanding debt, repay it.
```

Comment:
Why is this no longer needed? Won't debt in DAI terms still be present?

Reply:
since `sweepIntoDSR()` is called asap, there shouldn't be naked DAI in the contract.
since debt is set in DAI terms, the clearinghouse won't be able to repay DAI. Instead, if it is defunded in bulk, it will transfer sDAI (or any other erc20 that someone may send by mistake) directly.

Reply:
so I got rid of it to make the code shorter and save gas, although I don't think this function would be called at all

Reply:
Wouldn't the DAI debt in TRSRY still need to be adjusted? As during the `rebalance()` call the debt would have been set.

Reply:
yes, this is true... I'll modify the function so that it offsets the DAI debt if the defunded token is sDAI, good point.